packages/components/src/FormField/FormField.tsx

```diff
@@ -142,180 +147,179 @@ export interface FormFieldProps {
    * Callback to get the the status and message when validating a field
    * @param messages
    */
-  onValidation?(messages: ValidationProps[]): void;
+  onValidation?(messages?: ValidationProps[]): void;
 }
 
-export const FormField = React.forwardRef(
-  (
+export const FormField = React.forwardRef(FormFieldInternal);
+
+function FormFieldInternal(
+  {
+    align,
+    children,
+    defaultValue,
+    disabled,
+    errorMessage,
+    inline,
+    invalid,
+    max,
+    maxLength,
+    min,
+    name,
+    onFocus,
+    onBlur,
+    onChange,
+    onValidate,
+    onValidation,
+    placeholder,
+    readonly,
+    required,
+    rows,
+    size,
+    type = "text",
+    value,
+    validations = [],
+  }: FormFieldProps,
+  ref:
+    | Ref<HTMLInputElement>
+    | Ref<HTMLTextAreaElement>
+    | Ref<HTMLSelectElement>,
+) {
+  const [hasMiniLabel, setHasMiniLabel] = useState(
+    defaultValue || value ? true : false,
+  );
+  const identifier = uuid.v1();
+  const Wrapper = inline ? "span" : "div";
+
+  useEffect(() => {
+    /* DEPRECATED */
+    const status = errorMessage ? "fail" : "pass";
+    const message = errorMessage || "";
+    onValidate && onValidate(status, message);
+    /* END:DEPRECATED */
+
+    onValidation && onValidation(validations);
+  }, [value]);
+
+  if (required && validations) {
+    validations.push({
+      message: `${placeholder || "This"} is required`,
```

Comment:
Wondering if we can takes some lessons from things like this.

In this case suggesting a path forwards instead of pointing out a problem.

https://uxmovement.com/forms/how-to-make-your-form-error-messages-more-reassuring/

Reply:
I was actually just following an established jobber pattern http://sg-1.getjobber.com/components/shared/field-error/#writing-guidelines I'd pass it to the designers tho